db/migrate/20170719204710_create_pieces.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePieces < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pieces do |t|
+      t.integer :game_id
```

Comment:
@MelsenC Change the line I commented on to `t.integer :game_id, index: true`

Need to have the index in committed in migration file so that it will run on everyone elses machine (and production). Looks like you added it and then removed it (so it only worked on your machine).